src/Units/DoRUnits.java

```diff
@@ -0,0 +1,791 @@
+          moveType, actions, weapons, STAR_VALUE);
+    }
+  }
+
+  public static class AntiTankModel extends DoRUnitModel
+  {
+    private static final long serialVersionUID = 1L;
+    private static final int UNIT_COST = 11000;
+    private static final double STAR_VALUE = 1.4;
+    private static final int MAX_FUEL = 50;
+    private static final int IDLE_FUEL_BURN = 0;
+    private static final int MAX_AMMO = 6;
+    private static final int VISION_RANGE = 2;
+    private static final int MOVE_POWER = 4;
+
+    private static final MoveType moveType = new TiresRugged();
+    private static final UnitActionType[] actions = UnitActionType.COMBAT_VEHICLE_ACTIONS;
+    private static final WeaponModel[] weapons = { new DoRWeapons.AntiTankCannon() };
+
+    public AntiTankModel()
+    {
+      super("AntiTank", DoRUnitEnum.ANTITANK, UnitRoleEnum.SIEGE, ChassisEnum.TANK, UNIT_COST, MAX_AMMO, MAX_FUEL, IDLE_FUEL_BURN, VISION_RANGE, MOVE_POWER,
+          moveType, actions, weapons, STAR_VALUE);
+    }
+  }
+
+  public static class RocketsModel extends DoRUnitModel
+  {
+    private static final long serialVersionUID = 1L;
+    private static final int UNIT_COST = 15000;
+    private static final double STAR_VALUE = 1.4;
+    private static final int MAX_FUEL = 50;
+    private static final int IDLE_FUEL_BURN = 0;
+    private static final int MAX_AMMO = 5;
+    private static final int VISION_RANGE = 3;
+    private static final int MOVE_POWER = 5;
+
+    private static final MoveType moveType = new Tires();
+    private static final UnitActionType[] actions = UnitActionType.COMBAT_VEHICLE_ACTIONS;
+    private static final WeaponModel[] weapons = { new DoRWeapons.RocketRockets() };
+
+    public RocketsModel()
+    {
+      super("Rockets", DoRUnitEnum.ROCKETS, UnitRoleEnum.SIEGE, ChassisEnum.TANK, UNIT_COST, MAX_AMMO, MAX_FUEL, IDLE_FUEL_BURN, VISION_RANGE, MOVE_POWER,
+          moveType, actions, weapons, STAR_VALUE);
+    }
+  }
+
+  public static class MobileSAMModel extends DoRUnitModel
+  {
+    private static final long serialVersionUID = 1L;
+    private static final int UNIT_COST = 12000;
+    private static final double STAR_VALUE = 1.4;
+    private static final int MAX_FUEL = 50;
+    private static final int IDLE_FUEL_BURN = 0;
+    private static final int MAX_AMMO = 5;
+    private static final int VISION_RANGE = 5;
+    private static final int MOVE_POWER = 5; // Finally, sigh
+
+    private static final MoveType moveType = new Tires();
+    private static final UnitActionType[] actions = UnitActionType.COMBAT_VEHICLE_ACTIONS;
+    private static final WeaponModel[] weapons = { new DoRWeapons.MobileSAMWeapon() };
+
+    public MobileSAMModel()
+    {
+      super("Mobile SAM", DoRUnitEnum.MOBILESAM, UnitRoleEnum.ANTI_AIR, ChassisEnum.TANK, UNIT_COST, MAX_AMMO, MAX_FUEL, IDLE_FUEL_BURN, VISION_RANGE, MOVE_POWER,
+          moveType, actions, weapons, STAR_VALUE);
+    }
+  }
+
+  public static class RigModel extends DoRUnitModel
```

Comment:
This guy needs a Material, right?

Reply:
Sure, if and when we get around to temp air/ports.